[ADD] Recruitment: New refuse applicants doc (#9663)
odoo/documentation: content/applications/hr/recruitment/refuse_applicant.rst

=================
Refuse applicants
=================

> At any point in the recruitment process, an applicant can be refused for a job position. To refuse
> an applicant, click on an applicant's card to navigate to a detailed view of the card. At the top of

Reviewer: Since this is a new doc, I included some detailed navigational instructions on where the user should go (: Feel free to alter the instructions! I just took a guess on where they should go; as the HR expert, you would know better :blush: 
```suggestion
At any point in the recruitment process, an applicant can be refused for a job position. To refuse
an applicant, start by going to the :menuselection:`Recruitment app --> Applications --> All Applications`. On the :guilabel:`Applications` list, click the (undesired) applicant. At the top of
```

Author: Great idea- I added the other way to get to there (since there are two).

the applicant's card, there are several buttons. Click the one labeled :guilabel:`Refuse`.

.. _recruitment/refuse-reasons:

Refuse reasons
==============

> A :guilabel:`Refuse Reason` pop-up window appears. Click the desired :guilabel:`Refuse Reason` from

Reviewer: For skimmablility, I added a bit more context (:
```suggestion
Clicking :guilabel:`Refuse` makes the :guilabel:`Refuse Reason` pop-up window appear. Click the desired :guilabel:`Refuse Reason` from
```

are:

.. list-table::
   :header-rows: 1
   :stub-columns: 1

   * - Email Template
     - Refusal Reason
   * - :guilabel:`Recruitment: Refuse`
     - | :guilabel:`Doesn't fit the job requirements`
       | :guilabel:`Language issues`
       | :guilabel:`Role already fulfilled`
       | :guilabel:`Duplicate`
       | :guilabel:`Spam`
   * - :guilabel:`Recruitment: Not interested anymore`
     - | :guilabel:`Refused by Applicant: don't like job`
       | :guilabel:`Refused by Applicant: better offer`
       | :guilabel:`Refused by Applicant: salary`

Additional refusal reasons :ref:`can be created and existing ones can be modified or deleted
<recruitment/new-refuse>`.

.. note::
   Pre-configured email templates in Odoo sometimes use dynamic placeholders, which are customized
   pieces of data that populate dynamic content. For example, if the applicant's name is a piece of
   dynamic content, the applicant's name appears anytime that dynamic placeholder appears on the
   email template. For more detailed information on email templates, refer to the
   :doc:`../../general/companies/email_template` documentation.

> 

Reviewer: To complete the basic flow in this topmost section, I'm suggesting to bring the action to click the refuse button all the way up here, with the image. 

that way, readers who are just coming to the doc to see how applicants are refused can see the workflow right from the beginning, and additional features, such as configuring refuse reasons and the refusal email template, are found in later sections.

But, this is just a suggestion! Feel free to just ignore this comment, or reach out to me and we can have a discussion about it (:

```suggestion

After selecting a refuse reason, click the :guilabel:`Refuse` button to close the application.

.. image:: refuse_applicant/refuse.png
   :align: center
   :alt: An applicant's card with the refused banner appearing in the top-right corner in red.
```

.. _recruitment/new-refuse:

Create or modify refuse reasons
-------------------------------

> To view all currently configured refuse reasons, navigate to :menuselection:`Recruitment app -->

Reviewer: As a reader, I'm slightly confused because I'm still curious about one thing: why should I create a specific refuse reason? As a hiring manager, how does it improve my hiring process? I see that in the reporting, you can filter by refuse reason, so is the refuse reason

Instead, I'd modify the first sentence to quickly define what a refuse reason is, like:

```suggestion
Refuse reasons allow multiple recruiters to consistently document why a candidate was not a good fit, and tailor specific refusal reason email templates. To configure refuse reasons, navigate to :menuselection:`Recruitment app -->
```

P.S. this is my interpretation of what I *think* a refuse reason is. Feel free to alter it as you see fit!

> Settings --> Applications: Refuse Reasons`. All the refuse reasons appear in a list view.

Reviewer: ```suggestion
Configuration --> Applications: Refuse Reasons`. All the refuse reasons appear in a list view.
```

To create a new refuse reason, click the :guilabel:`New` button in the top-left corner. A blank
line appears at the bottom of the :guilabel:`Description` column.

Type in the new refuse reason in the field. It is recommended to enter a reason that is short and
concise, such as `offer expired` or `withdrew application`.

Then, in the :guilabel:`Email Template` field, click on the field to reveal a drop-down menu.
Select an :guilabel:`Email Template` from the list to be used when this refuse reason is selected.

If a new :guilabel:`Email Template` is desired, type in the name for the new template in the field.
Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Email Template` form pop-up
window appears.

Enter a :guilabel:`Name` for the form and an email :guilabel:`Subject` in the corresponding fields.
Enter the email content in the :guilabel:`Content` tab. Make any other desired modifications to the
template, then click :guilabel:`Save & Close` to save the template and return to the
:guilabel:`Refuse Reasons` list. The new template appears in the new refuse reason :guilabel:`Email
Template` field.

> To make any modifications to a refuse reason, click on the line, and make any desired changes.

Reviewer: ```suggestion
To make modifications to an existing refuse reason, click the line, and make any desired changes.
```

Send refusal email
==================

> After a :ref:`Refuse Reason <recruitment/refuse-reasons>` is selected, two fields appear below the

Reviewer: For skimmability, I wanted to include more details for the reader about where we are in the workflow
```suggestion
After clicking the :guilabel:`Refuse` button on an applicant form, a :ref:`Refuse Reason <recruitment/refuse-reasons>` can be selected. Then, two fields appear below the
```

:guilabel:`Send Email` checkbox.

The email template associated with the refusal reason populates the :guilabel:`Email Template`
field. If a different email template is desired, select a different template from the
:guilabel:`Email Template` drop-down menu. To view the email template, click the
:icon:`fa-external-link` :guilabel:`Internal link` icon to the right of the :guilabel:`Email
Template` field. The email template loads in a :guilabel:`Open: Email Template` pop-up window, and
can be modified if needed. After making modifications, click :guilabel:`Save & Close`.

To send the refusal email to the applicant, click :guilabel:`Refuse`. The refusal email is sent to
the applicant, and a red :guilabel:`Refused` banner appears on the applicant's card in the top-right
corner.

.. image:: refuse_applicant/refuse.png
   :align: center
   :alt: An applicant's card with the refused banner appearing in the top-right corner in red.

View refused applicants
=======================

After refusal, the applicant's card is no longer visible in the job position's Kanban view. However,
it is still possible to view applicant's who have been refused.

Job position view
-----------------

> To view only the refused applicants for that job position, click the :icon:`fa-caret-down`

Reviewer: how do we navigate to the job position's kanban view, again? :thinking:

.. _recruitment/new-refuse:

Create or modify refuse reasons
-------------------------------

Refuse reasons allow recruiters to document why an applicant was not a good fit, and send specific
refusal reason email templates to the applicant.

To view and configure refuse reasons, navigate to :menuselection:`Recruitment app --> Configuration
--> Applications: Refuse Reasons`. All the refuse reasons appear in a list view.

To create a new refuse reason, click the :guilabel:`New` button in the top-left corner. A blank
line appears at the bottom of the :guilabel:`Description` column.

Type in the new refuse reason in the field. It is recommended to enter a reason that is short and
concise, such as `offer expired` or `withdrew application`.

Then, in the :guilabel:`Email Template` field, click on the field to reveal a drop-down menu.
Select an :guilabel:`Email Template` from the list to be used when this refuse reason is selected.

If a new :guilabel:`Email Template` is desired, type in the name for the new template in the field.
Then, click :guilabel:`Create and edit...` and a :guilabel:`Create Email Template` form pop-up
window appears.

> Enter a :guilabel:`Name` for the form and an email :guilabel:`Subject` in the corresponding fields.

Author: I don't see that field- I just see the fields in the image in the doc. I checked other runbots as well- I don't see that anywhere.

Enter the email content in the :guilabel:`Content` tab. Make any other desired modifications to the
template, then click :guilabel:`Save & Close` to save the template and return to the
:guilabel:`Refuse Reasons` list. The new template appears in the new refuse reason :guilabel:`Email
Template` field.

To make modifications to an existing refuse reason, click on the line, and make any desired changes.

.. _recruitment/email:

> Send refusal email

Author: I thought about that- but I did adda sentence after the seciton you mentioned, and there is a link in the text to jump to the send email section. So because of that, I think it's ok- they can go in order (just with a click).
